contracts/rewards/RewardsManagerStorage.sol

```diff
@@ -8,7 +8,7 @@ import "../governance/Managed.sol";
 contract RewardsManagerV1Storage is Managed {
     // -- State --
 
-    uint256 public issuanceRate;
+    uint256 public issuanceRateDeprecated;
```

Comment:
Small optimization: we can make this variable private to avoid including a getter in the bytecode